.eslintrc

```diff
@@ -11,8 +11,9 @@
   "overrides": [
     {
       "files": [
-        "packages/grafana-ui/**/*/!(*.story).{ts,tsx}",
-        "packages/jaeger-ui-components/**/*.{ts,tsx,js}",
+        "packages/grafana-ui/src/components/uPlot/**/*.{ts,tsx}",
+        "packages/grafana-ui/src/components/GraphNG/GraphNG.tsx",
+        "packages/grafana-ui/src/components/BarChart/BarChart.tsx",
```

Comment:
GraphNG and BarChart will shortly say good bye to hooks anyways :) https://github.com/grafana/grafana/pull/30941